study-builder/studies/testboston/longitudinal-covid-v2.conf

```diff
@@ -0,0 +1,252 @@
+              """
+            },
+            {
+              "question": ${_includes.baseline_covid.covid_test_result} {
+                "stableId": ${id.q.longitudinal_covid_test_result}
+              },
+              "blockType": "QUESTION",
+              "shownExpr": """
+                user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_COVID_BEEN_TESTED"].answers.hasOption("YES")
+                && user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_COVID_TEST_KIND"].answers.hasOption("NOT_SURE")
+              """
+            },
+            {
+              "question": ${_includes.longitudinal.hospitalized},
+              "blockType": "QUESTION",
+              "shownExpr": """
+                user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_COVID_BEEN_TESTED"].answers.hasOption("YES")
+                && user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_COVID_TEST_KIND"].answers.hasOption("NOT_SURE")
+                && user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_COVID_TEST_RESULT"].answers.hasOption("POS")
+              """
+            },
+            {
+              "question": ${_includes.longitudinal.hospital_name},
+              "blockType": "QUESTION",
+              "shownExpr": """
+                user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_COVID_BEEN_TESTED"].answers.hasOption("YES")
+                && user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_COVID_TEST_KIND"].answers.hasOption("NOT_SURE")
+                && user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_COVID_TEST_RESULT"].answers.hasOption("POS")
+                && user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_HOSPITALIZED"].answers.hasTrue()
+              """
+            },
+            {
+              "question": ${_includes.longitudinal.hospital_days},
+              "blockType": "QUESTION",
+              "shownExpr": """
+                user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_COVID_BEEN_TESTED"].answers.hasOption("YES")
+                && user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_COVID_TEST_KIND"].answers.hasOption("NOT_SURE")
+                && user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_COVID_TEST_RESULT"].answers.hasOption("POS")
+                && user.studies["testboston"].forms["LONGITUDINAL_COVID"].instances[specific].questions["LONGITUDINAL_HOSPITALIZED"].answers.hasTrue()
+              """
+            },
```

Comment:
This set of lines is the new stuff. Rest of file is a copy of `longitudinal-covid.conf` file. I created a new v2 conf file to make it easier to work with.

Reply:
In the new code showExpr: we explicitly checking for condition :
["LONGITUDINAL_COVID_TEST_KIND"].answers.hasOption("NOT_SURE")
We don't want to ask Hospitalized questions if TEST_KIND is "VIRAL_TEST or BLOOD_TEST" ?

Reply:
As far as I know, it only shows up for `NOT_SURE`. This is what the attached document in the ticket shows as well.

Reply:
ok